code/datums/abilities/wizard.dm

```diff
@@ -299,7 +299,7 @@
 		object.owner = src
 
 	castcheck()
-		return !istype(src, /datum/targetable/spell/prismatic_spray/admin) && holder.owner.wizard_castcheck(src)
+		return holder.owner.wizard_castcheck(src)
```

Comment:
Removing this is good but shouldn't you reintroduce this behaviour back in `/datum/targetable/spell/prismatic_spray/admin`'s castcheck? (Basically just overriding it to `return TRUE`?)

Reply:
I thought about it, but, frankly, it seemed unnecessary in hindsight

Reply:
[Having it be robeless, that is]

Reply:
Because we already have a way to bypass that through the arcane_power bioeffect (with variant set to 2)